ts/store/storages/keychain.ts

```diff
@@ -0,0 +1,29 @@
+import * as Keychain from "react-native-keychain";
+import { Storage } from "redux-persist";
+
+/**
+ * A storage to save/restore values from/to the Keychain.
+ * The react-native-keychain library do not support storing multiple key/value so we
+ * use the service as key. Each value is stored as a new service and a constant username
+ * is set.
+ */
+export const USERNAME = "REDUX_PERSIST";
+
+export default function createSecureStorage(): Storage {
```

Comment:
can we have a test for this? (by mocking `Keychain`)

Reply:
Yes i am goin to add tests.

Reply:
Mocked Keychain and test added.